scrape.py

```diff
@@ -37,19 +38,27 @@ def fetch(url):
     soup = BeautifulSoup(r.content, 'html.parser')
     user_info = soup.find_all('div', attrs={'class': 'stat'})
 
+    t = Texttable()
+    info_list = []
```

Comment:
`[]` already suggest that its a list. Let's call it `profile_stats` or something like that